ControlMode control_mode
InputMode input_mode
float32 input_pos
float32 input_vel
float32 input_torque
